libflux/go/libflux/bench_test.go

```diff
@@ -88,11 +88,11 @@ func ParseReturnJSON(fluxFile string) error {
 func ParseAndDeserializeJSON(fluxFile string) error {
 	p := libflux.Parse(fluxFile)
 	defer p.Free()
-	bs, err := p.MarshalJSON()
```

Comment:
I left this test case not calling `buf.Free()` so we can test the finalizer.

Reply:
Ok, I'm not sure if there's a way to test the finalizer in Go though, since my understanding is that the finalizer is not guaranteed to be called if the process exits.  Not sure there is a reliable way to do it.  I wonder if the `-race` flag does anything in this regard?